.github/workflows/gettext.yml

```
@@ -0,0 +1,29 @@
+name: Gettext Updates
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/elementary/docker:next-unstable
+
+    steps:
+    - name: Install git
+      run: |
+        apt-get update
+        apt-get install git -y
+
+    - name: Clone repository
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GIT_USER_TOKEN }}
+
+    - name: Update Translation Files
+      uses: elementary/actions/gettext-template@next
```

Don't need to install git with the one from main :)

```suggestion
    steps:
    - name: Clone repository
      uses: actions/checkout@v4
      with:
        token: ${{ secrets.GIT_USER_TOKEN }}

    - name: Update Translation Files
      uses: elementary/actions/gettext-template@main
```

---

Done